cmd/agent/install_script.sh

```diff
@@ -188,13 +188,10 @@ Please follow the instructions on the Agent setup page:
     exit;
 fi
 
-printf "\033[31mRefreshing symlink...\n\033[0m\n"
-$sudo_cmd ln -sf /opt/datadog-agent/bin/agent/agent /usr/bin/datadog-agent
-
 if [ $dd_upgrade ]; then
   if [ -e $LEGACY_CONF ]; then
     # try to import the config file from the previous version
-    icmd="datadog-agent import $LEGACY_ETCDIR $ETCDIR"
+    icmd="/opt/datadog-agent/bin/agent/agent import $LEGACY_ETCDIR $ETCDIR"
```

Comment:
why change this? (the symlink `/usr/bin/datadog-agent -> /opt/datadog-agent/bin/agent/agent` should be created by the package since `beta.2`)

Reply:
Was not aware of this, reverting :)